pkg/aerospike/aerospike.go

```diff
@@ -9,32 +9,48 @@ import (
 	"github.com/aerospike/aerospike-client-go"
 )
 
-func (m *AerospikeConnection) BuildContext(cx model.CheckContext) {
-	m.Host = cx.Host
-	m.Port = cx.Port
+type checker struct {
+	host string
+	port int
 }
 
-func (m *AerospikeConnection) Validate() error {
-	if len(m.Host) == 0 {
+// NewChecker creates a new checker
```

Comment:
This document is the description for every checker. What I did (using the same copy/paste function with the same description) was because of the live coding and it was wrong. can we have a complete sentence here? something like `NewChecker creates a new checker for Aerospike, AeroSpike is an enterprise nosql platform can be found at https://www.aerospike.com/` (this is an example and I don't nothing about aerospike :D

Reply:
Yeah, I understand documentation how much is important in Go. I just wanted this PR to be as simple as possible. We can do further PRs with the purpose of documentation. Many things may have gone wrong in the project and we should fix all issues step by step.